rosco-core/src/main/groovy/com/netflix/spinnaker/rosco/providers/CloudProviderBakeHandler.groovy

```diff
@@ -77,10 +77,11 @@ abstract class CloudProviderBakeHandler {
 
       if (ami_name) {
         bakeKey += ":$ami_name"
-      } else {
-        bakeKey += ":$package_name"
       }
 
+      String packages = package_name.tokenize().join('-')
```

Comment:
The only issue I see here is that there is a potential naming-collision. These two bake requests would result in the same token for the `packages` segment of the bake key:
`package_name == 'a-b c'`
`package_name == 'a b-c'`

Can avoid this (or at least greatly reduce the likelihood of it occurring) by choosing as the delimiter a character that is unlikely to appear in a package name.